docs/release-notes.asciidoc

```diff
@@ -20,6 +20,62 @@ required to enable the UI.
 * <<post-upgrade-req, Enable the process analyzer>>. This is only required if you want to view
 <<alerts-analyze-events, graphical representations of process relationships>>.
 
+[discrete]
+[[release-notes-7.10.0]]
+== 7.10.0
+
+[discrete]
+[[breaking-changes-7.10.0]]
+==== Breaking changes
+
+*Signals template updated for rollover indices*
+
+The `create_index_route` now checks if the template needs to be upgraded
+before creating the index. If the index already exists and the template was upgraded,
+the index rolls over so that the write index has the upgraded mapping.
+This breaks the old mappings that have `risk_score mapped` as a keyword.
+In the new mapping, `signal.rule.risk_score` is a float.  After rolling over,
+there is a conflict between the old and new `signal.rule.risk_score` for some
+features, such as aggregations.
```

Comment:
Would it make sense to call out the need to add the `view_index_metadata` permission here, as that's the change users will need to make for Event Correlation rules to function upon upgrade?

Reply:
Add a note here and on the detections requirement page